### ISSUE TITLE
Unable to load weights

### ISSUE THREAD

**Issue by PrakharPrashantSingh** [facebookresearch/detectron2#4151]
i am using the official github repo, downloaded and setup detectron2 successfully but it is not able to load ".pkl" file, one solution that i found out is to download these pkl file and run, its working for "test_windows_

install.py" file 

but i am not able to download other weights like-  detectron2://COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x/137849600/model_final_f10217.pkl

what to do in this scenario, i am on windows 10

Thanks

error output-   File "C:\Users\Prashant\Anaconda3\envs\detectron\lib\site-packages\fvcore\common\checkpoint.py", line 153, in load
    assert os.path.isfile(path), "Checkpoint {} not found!".format(path)
AssertionError: Checkpoint detectron2://ImageNetPretrained/MSRA/R-50.pkl not found!

**Comment by PrakharPrashantSingh** [comment]
hi,

please look into this ...

command used->
python demo.py --config-file ../configs/COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml --input capture.PNG --opts MODEL.WEIGHTS detectron2://COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x/137849600/model_final_f10217.pkl

error output->
assert os.path.isfile(path), "Checkpoint {} not found!".format(path)
AssertionError: Checkpoint detectron2://COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x/137849600/model_final_f10217.pkl not found!

so basically program is not able to load the model pkl file, i am on windows, what to do in this scenario..

Thanks

**Comment by ppwwyyxx** [comment]
> You've chosen to report an unexpected problem or bug. Unless you already know the root cause of it, please include details about it by filling the [issue template](https://github.com/facebookresearch/detectron2/issues/new/choose).
The following information is missing: "Instructions To Reproduce the Issue and Full Logs"; "Your Environment";

**Comment by zsj8991** [comment]
Hi,I run this command,then run 'demo' successfully

pip install fvcore==0.1.1.dev200512

**Comment by PrakharPrashantSingh** [comment]
> Hi,I run this command,then run 'demo' successfully
> 
> pip install fvcore==0.1.1.dev200512

yes bro, you were right.. it worked!! 

thanks

**Comment by iwannabewater** [comment]
> Hi,I run this command,then run 'demo' successfully
> 
> pip install fvcore==0.1.1.dev200512

thanks bro！！

**Comment by gt8mar** [comment]
Detectron2 now requires fvcore>=0.1.2, which version should I install now?
